recipe/meta.yaml

```diff
@@ -34,15 +34,18 @@ test:
     - colorcet
 
 about:
-  home: http://github.com/pyviz/colorcet
+  home: https://colorcet.holoviz.org/index.html
   license: CC-BY-4.0
+  license_family: CC
+  license_url: https://github.com/holoviz/colorcet/blob/master/LICENSE.txt
   license_file: LICENSE.txt
   summary: Collection of perceptually uniform colormaps
   description: |
     colorcet is a collection of perceptually uniform colormaps for use with
     Python plotting programs like bokeh, matplotlib, holoviews, and datashader.
-  doc_url: https://pypi.python.org/pypi/colorcet
-  doc_source_url: https://github.com/bokeh/colorcet/blob/master/README.md
+  dev_url: https://github.com/pyviz/colorcet
```

Comment:
This redirects to `https://github.com/holoviz/colorcet`. I think it would be best to change to `holoviz` as it appears to be a part of that suite of tools after a reorganization.

Reply:
good catch! thank you